lib/sugar.js

```diff
@@ -10,16 +10,23 @@ if (typeof navigator !== 'undefined') {
   var sugarApiClient = new JSONAPIClient(config.sugarHost, {
     'Content-Type': 'application/json',
     'Accept': 'application/json',
+  }, {
+    beforeEveryRequest: function() {
+      auth.checkBearerToken()
+        .then(function () {
+          return null;
+        });
+    }
```

Comment:
This mixin inclusion crashes PFE when accessing a project home page.

Reply:
Good catch. I missed out a `return`.